README.md

```diff
@@ -40,6 +18,12 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
 
 **Use GOV.UK Docker to run any commands that follow.**
 
+### Building the schemas
```

Comment:
Optional: since this is a non-standard section, it would be helpful to explain when someone should run this command e.g. to get validate a change to the schemas, without running the tests (which are separate).